server/src/main/java/org/elasticsearch/action/bulk/BulkResponse.java

```diff
@@ -134,7 +134,6 @@ public void writeTo(StreamOutput out) throws IOException {
         out.writeZLong(ingestTookInMillis);
     }
 
-    @Override
     public RestStatus status() {
```

Comment:
Indeed, but I worry that we won't do that follow-up. Could we fix these places up first in a preliminary PR so that this PR doesn't leave any `status()` methods that just always return `OK`?